src/CMakeLists.txt

```diff
@@ -173,8 +173,7 @@ target_include_directories(exiv2lib_int PUBLIC
 if (EXIV2_ENABLE_WEBREADY)
 
     if( EXIV2_ENABLE_SSH )
-        target_include_directories(exiv2lib SYSTEM PUBLIC ${LIBSSH_INCLUDE_DIR})
-        target_link_libraries(exiv2lib PUBLIC ${LIBSSH_LIBRARIES})
+        target_link_libraries(exiv2lib PUBLIC ssh)
```

Comment:
I see a problem with this. With this change CMake will try to always link with the `ssh` library available in the system, and not the one you specify to CMake you want to use (in case you give the proper paths to sue a specific version).

Reply:
If you feel you must do that, you can disable finding libssh via cmake.